test/e2e/README.md

```diff
@@ -60,6 +64,28 @@ T_TINKERBELL_INVENTORY_CSV # path to hardware-inventory.csv file
 T_TINKERBELL_SSH_AUTHORIZED_KEY # ssh public key for connectioning to machines
 ```
 
+# CloudStack tests requisites
+
+The following env variables need to be set:
+```
+T_CLOUDSTACK_DOMAIN
+T_CLOUDSTACK_ZONE
+T_CLOUDSTACK_ACCOUNT
+T_CLOUDSTACK_NETWORK
+T_CLOUDSTACK_MANAGEMENT_SERVER
+T_CLOUDSTACK_SSH_AUTHORIZED_KEY
+T_CLOUDSTACK_TEMPLATE_REDHAT_1_20
+T_CLOUDSTACK_TEMPLATE_REDHAT_1_21
+T_CLOUDSTACK_COMPUTE_OFFERING_LARGE
+T_CLOUDSTACK_COMPUTE_OFFERING_LARGER
+T_CLOUDSTACK_CONTROL_PLANE_HOST
+T_CLOUDSTACK_POD_CIDR
+T_CLOUDSTACK_SERVICE_CIDR
+
+EKSA_CLOUDSTACK_B64ENCODED_SECRET (when running against local eks-a)
```

Comment:
Why is this documented as such? We don't need this set for CI environment?

Reply:
Yes, will change documentation. This is a required var for running eksa with cloudstack